ProQuest.js

```diff
@@ -499,6 +517,61 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	
+	for(var i = 0; i < refs.length; i++) {
+		var ref = refs[i];
+		var hiddenData = ZU.xpathText(ref.html, './span');
+		var visibleData = ZU.xpathText(ref.html.getElementsByClassName('results_list_copy'), './text()').trim().split(';').reverse();
+		
+		var item = new Zotero.Item("book");
+
+		var date = /\(([\w\d\s]+)\)/.exec(visibleData[0]);
+		var place = /([\w,\s]+)\(/.exec(visibleData[0]);
+		var isbn = /isbn,\svalue\s=\s\'(\d+)\'\n/i.exec(hiddenData);
+		var language = /language_code,\svalue\s=\s\'([A-Za-z]+)\'\n/i.exec(hiddenData);
+		var numPages = /page_count,\svalue\s=\s\'(\d+)\'\n/i.exec(hiddenData);
+		var locNum = /lccn,\svalue\s=\s\'([\d\s\-\.\w]+)\'\n/i.exec(hiddenData);
```

Comment:
\d is in \w, `.` does not need escaping inside `[]`. Tip: you don't need to escape `-` inside `[]` if you put it first.